exercises/go-counting/go_counting_test.py

```diff
@@ -0,0 +1,91 @@
+import unittest
+import gocounting
+
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.1.0
+
+board5x5 = "\n".join([
+    "  B  ",
+    " B B ",
+    "B W B",
+    " W W ",
+    "  W  "
+])
+
+board9x9 = "\n".join([
+    "  B   B  ",
+    "B   B   B",
+    "WBBBWBBBW",
+    "W W W W W",
+    "         ",
+    " W W W W ",
+    "B B   B B",
+    " W BBB W ",
+    "   B B   "
+])
+
+
+class GoCountingTest(unittest.TestCase):
+    def test_5x5_for_black(self):
+        board = gocounting.GoCounting(board5x5)
```

Comment:
class `GoCounting` should not be the same name as the module `gocounting`. Please rename to something more intuitive, such as `Board`